### ISSUE TITLE
GREI 4: Task 4: Implement FAIR Signposting in GREI repositories.

### ISSUE THREAD

**Issue by cmbz** [IQSS/dataverse-pm#138]

**Comment by pdurbin** [comment]
Signposting was added to Dataverse in 5.14:

- https://github.com/IQSS/dataverse/pull/8981

**Comment by cmbz** [comment]
Yes, thanks, I'll add to the (soon to be added) related section. This issue is here to track the upcoming, related GREI year 3 work as well. Stay tuned. :)
